### PR TITLE
HLT menu for XeXe MC production (94X)

### PR DESCRIPTION
**Pull request by silviodonato**
Adding HLT XeXe frozen menu to be used in the XeXe MC production.
The menu works with L1Menu_Collisions2017_v3_m6_xml.

92X PR: #21316 
Since this menu will be only used for the XeXe MC production, there is no 100X PR.

### CODE REVIEW

**Comment by silviodonato** [comment]
please test

File: HLTrigger/Configuration/test/getFrozenHLT.sh [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #! /bin/bash
 
 # ConfDB configurations to use
-TABLES="Fake Fake1 Fake2 2e34v22 2e34v31 2e34v40"
+TABLES="XeXe Fake"
+#Fake Fake1 Fake2 2e34v22 2e34v31 2e34v40
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/test/getFrozenHLT.sh]: Can you please restore the other frozen menus?

**Comment by silviodonato** [comment]
please test

**Comment by mandrenguyen** [comment]
Thanks @silviodonato !  Would it make sense to activate this in the XeXe MC wf (148) in the same PR?  It's currently running HLT:@fake2

**Comment by silviodonato** [comment]
@mandrenguyen which line should I change?

**Comment by mandrenguyen** [comment]
steps['DIGIHI2017']=merge([{'-s':'DIGI:pdigi_hi,L1,DIGI2RAW,HLT:@fake2'}, hiDefaults2017, step2Upg2015Defaults])
Configuration/PyReleaseValidation/python/relval_steps.py

I guess you also want to change this line, to grab the new GT you've created with the proper L1
hiAlca2017 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017_pp_on_XeXe'}

**Comment by silviodonato** [comment]
What is the new GT? How is it called? (I'm not a HI expert...)

**Comment by mandrenguyen** [comment]
Sorry Silvio, I stated that confusingly.  You should simply use the autoconditions that you are adding in this PR in wf 148.  In this case, that won’t change the GT

**Comment by silviodonato** [comment]
In the cmsDriver command used in the MC production you can directly use "HLT:XeXe" . 
Usually autocond is not used for the MC production, as far as I know. 
To run this HLT menu you need a GT containing L1Menu_Collisions2017_v3_m6_xml.
Are you involved in the MC production for XeXe?
If I can make your life simpler adding a line in the PR, I'm happy to do it but please tell me explicitly what you want.

**Comment by mandrenguyen** [comment]
Indeed I was suggesting to switch to HLT:XeXe
I had in mind that we would not have to switch to a dedicated GT for the L1 menu in this case.
For XeXe we simply recorded MB triggers using an L1 seed that is present in GT pointed by the autoconditions already used by this workflow.  
However, when I tested your PR with HLT:XeXe, I see that I neglected to consider the calibration triggers.  HLT_HIHcalNZS_v1 is crashing, looking for  L1_DoubleJet40er3p0

It looks like this is due to the fact that the autocond used here points to a GT with L1Menu_Collisions2017_v4, and this L1 path was removed between v3 and v4. 

Perhaps it's best to just move ahead with the unmodified PR.   Thanks for bearing with me.  Do you happen to know if it's possible to customize an L1 menu from driver?

**Comment by silviodonato** [comment]
+1

**Comment by silviodonato** [comment]
When @yimuchen  and @zc11 asked me to add the XeXe menu, I explicitly asked them to provide a GT containing L1Menu_Collisions2017_v3. They told me that it was already in place.

**Comment by silviodonato** [comment]
@mandrenguyen  I think you have to write a customize function changing the GlobalTag:

`process.GlobalTag.toGet.append(
  cms.PSet(
    record = cms.string("L1TUtmTriggerMenuRcd"),
    tag = cms.string("L1Menu_Collisions2017_v3_m6_xml")),
)`

and then call it with something like
` cmsDriver .... --customise=your/folder/yourCustomizeFile.yourCustomizeFunction `

**Comment by davidlange6** [comment]
merge

**Pull request merged**